universal-login-contracts/test/contracts/ERC1077.js

```diff
@@ -202,6 +202,59 @@ describe('ERC1077', async  () => {
     });
   });
 
+  describe('Multi-Signature call', async () => {
```

Comment:
We should split into two describe: `Multi-Signature call with 2 signatures` and `Multi-Signature call with 3 signatures`.